docassemble/ALKilnTests/data/sources/languages.feature

```diff
@@ -0,0 +1,52 @@
+@langs
+Feature: Multi-language tests
+
+# Usually only run default test and skip or shorten others
+# When commanded, run all non-default language tests
+
+# We'll need to test these manually each time, as our users would,
+# and check that the automatic tests didn't trigger the language
+# tests
+
+# Not sure how to test urls like this: http://es.<other stuff>interview.yml
+# That's how wikipedia does it.
```

Comment:
Wikipedia has a very different infrastructure than docassemble; I doubt that if some one is running a multi-lingual interview, they would have different copies of the interview on different servers. which they would need to do with `es.<other stuff>interview.yml`, so I don't think that needs to be noted in the file.

The other way that I would recommend testing or supporting (eventually at least) is the `Accept-Language` header that [docassemble uses if there isn't a `lang` in the URL](https://docassemble.org/docs/language.html#configuration):

> If there is no lang parameter in the URL, docassemble will look for an Accept-Language header in the request from the user’s browser. This is typically set by the user’s web browser to whatever language is defined in the web browser settings. So if the user is a German speaker who is using a web browser that is set up to use the language de, then the language de will be used for translations. In interviews, you can read this language by calling language_from_browser().